google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -472,10 +472,9 @@ def dataframe_to_parquet(dataframe, bq_schema, filepath, parquet_compression="SN
             columns in the DataFrame.
         filepath (str):
             Path to write Parquet file to.
-        parquet_compression (str):
-            (optional) The compression codec to use by the the
-            ``pyarrow.parquet.write_table`` serializing method. Defaults to
-            "SNAPPY".
+        parquet_compression (optional[str]):
```

Comment:
```suggestion
        parquet_compression (Optional[str]):
```
(nit) Should be uppercased for consistency.